homeassistant/components/tradfri/coordinator.py

```diff
@@ -0,0 +1,138 @@
+"""Tradfri DataUpdateCoordinator."""
```

Comment:
If we don't want to add more tests for the exception handling we should ignore the coordinator module in `.coveragerc`.

Reply:
I was not planning on adding anymore tests in this PR so I'll update it.